docs/notebooks.rst

```diff
@@ -0,0 +1,122 @@
+.. _Jupyter Notebooks:
+
+#################
+Jupyter Notebooks
+#################
+
+Please see the :ref:`Development Procedure` section for notebook development guidelines.
+
+.. seealso::
+
+    For detailed instructions on formatting notebook, please visit the
+    `STScI notebook style guide. <https://github.com/spacetelescope/style-guides/blob/master/guides/jupyter-notebooks.md>`_
+
+
+Installation
+************
+
+For instructions on how to install Jupyter notebooks, please visit the
+`Jupyter Installation page <https://jupyter.org/install>`_.
+
+
+We also strongly recommend using `Conda <https://docs.conda.io/projects/conda/en/latest/index.html>`_ for
+managing environments. If you need instructions on how to setup Conda, please see the Conda's
+`Getting Started <https://docs.conda.io/projects/conda/en/latest/user-guide/getting-started.html>`_ documentation.
```

Comment:
This might be opening a can of worms, and if @camipacifici in particular wants to say :see_no_evil: :hear_no_evil: we can discuss this at a later time (i.e., not part of this PR), but I think we should consider softening this recommendation at this point.  I personally think `venv` + `pip` is now superior to `conda` as an environment-management solution. Of course opinions may vary on this, but perhaps we should soften this a bit to something like:

```suggestion
We also strongly recommend using an environment managing tool for keeping track of dependencies and
preventing conflicts.  Options include the `venv` package and `pip` tools in the python standard library, or the 
 `Conda <https://docs.conda.io/projects/conda/en/latest/index.html>`_ tool combined with the `conda-forge`
 package index. If you need instructions on how to setup Conda, please see the Conda's
`Getting Started <https://docs.conda.io/projects/conda/en/latest/user-guide/getting-started.html>`_ documentation.
```